flexget/plugins/input/listdir.py

```diff
@@ -31,7 +31,12 @@ def on_task_input(self, task, config):
         entries = []
         for folder in config:
             folder = path(folder).expanduser()
-            for filepath in folder.listdir():
+            try:
+                dir_files = folder.listdir()
+            except OSError as e:
+                log.error('error accessing folder: %s' % e.strerror)
```

Comment:
Error is already logged as a level so starting message with it redundant. Also all logged messages are capitalized.

Change this to 'Path %s can not be accessed' or something like that.